app/controllers/api/storage.php

```diff
@@ -5,6 +5,7 @@
 use Appwrite\Event\Audit;
```

Comment:
Let's make sure that whatever changes we implement on the database controller we also implement on storage.